website/docs/Miscellaneous/mdx.md

```diff
@@ -14,3 +14,5 @@ export const Highlight = ({children, color}) => ( <span style={{
 
 <Highlight color="#25c2a0">Docusaurus green</Highlight> and <Highlight color="#1877F2">Facebook blue</Highlight> are my favorite colors.
 I can write **Markdown** alongside my _JSX_!
```

Comment:
Am adding this as a counter-point on an earlier line...

Comment is about an inappropriate use of exclamation marks in markdown.  Too emotional...